lxd/db/cluster/config.mapper.go

```diff
@@ -27,7 +27,7 @@ const configDelete = `DELETE FROM %s_config WHERE %s_id = ?`
 
 // GetConfig returns all available config.
 // generator: config GetMany
-func GetConfig(ctx context.Context, tx *sql.Tx, parent string) (map[int]map[string]string, error) {
+func GetConfig(ctx context.Context, tx *sql.Tx, parent string, filters ...ConfigFilter) (map[int]map[string]string, error) {
```

Comment:
@masnax I can't see where this function errors out if an empty filter is provided?